android/rxJava/app/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    defaultConfig {
+        applicationId "com.sfsu.ajinkyachalke.rxjavatutorial"
```

Comment:
Please change package name to org.androidtutorials.rxjava. Also refactor all Java package names.

Reply:
I have made the changes:
1. Added slides in .pptx and .pdf format under doc/ of my application's root folder
2. Changed the package name to org.androidtutorials.rxjava. Rebuilt and tested the code.
3. Squashed commits into one